### PR TITLE
feat(judicial-system): Enable court users to upload case files to Auður from overview screen

### PR DESCRIPTION
**Pull request by oddsson**
# Enable court users to upload case files to Auður from overview screen

https://app.asana.com/0/1199153462262248/1201481159264532/f

## What

Enable court users to upload case files to Auður from overview screen

## Why

Today, users can only upload case files to Auður when a case has been ruled in. Court users need to be able to do this sooner in the process.

## Screenshots / Gifs

Screenshot 2021-12-14 at 10 24 09

## Checklist:

- [x] I have performed a self-review of my own code
- [ ] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] Formatting passes locally with my changes
- [x] I have rebased against main before asking for a review

### CODE REVIEW

File: apps/judicial-system/backend/src/app/modules/file/guards/test/viewCaseFileGuard.spec.ts [hunk 1]
```diff
@@ -0,0 +1,239 @@
...
+    let then: Then
+
+    beforeEach(() => {
+      mockRequest.mockImplementationOnce(() => ({ user, case: theCase }))
+
+      then = givenWhenThen()
+    })
+
+    it('should activate', () => {
+      console.log(then)
```
Comment (oddsson) [inline apps/judicial-system/backend/src/app/modules/file/guards/test/viewCaseFileGuard.spec.ts:55]: Please consider removing this

File: apps/judicial-system/backend/src/app/modules/file/guards/test/viewCaseFileGuard.spec.ts [hunk 2]
```diff
@@ -0,0 +1,239 @@
...
+      let then: Then
+
+      beforeEach(() => {
+        mockRequest.mockImplementationOnce(() => ({ user, case: theCase }))
+
+        then = givenWhenThen()
+      })
+
+      it('should activate', () => {
+        console.log(then)
```
Comment (oddsson) [inline apps/judicial-system/backend/src/app/modules/file/guards/test/viewCaseFileGuard.spec.ts:82]: Please consider removing this

File: apps/judicial-system/backend/src/app/modules/file/guards/test/viewCaseFileGuard.spec.ts [hunk 3]
```diff
@@ -0,0 +1,239 @@
...
+    let then: Then
+
+    beforeEach(() => {
+      mockRequest.mockImplementationOnce(() => ({ user, case: theCase }))
+
+      then = givenWhenThen()
+    })
+
+    it('should activate', () => {
+      console.log(then)
```
Comment (oddsson) [inline apps/judicial-system/backend/src/app/modules/file/guards/test/viewCaseFileGuard.spec.ts:129]: Please consider removing this

File: apps/judicial-system/backend/src/app/modules/file/guards/test/viewCaseFileGuard.spec.ts [hunk 4]
```diff
@@ -0,0 +1,239 @@
...
```
Comment (oddsson) [inline apps/judicial-system/backend/src/app/modules/file/guards/test/viewCaseFileGuard.spec.ts:147]: Please consider removing this

**Review by oddsson — COMMENTED** [review]
There are a few console.logs in one of the test files.

**Review by gudjong — APPROVED** [review]

**Pull request merged**